tests/tamr_client/test_transformations.py

```diff
@@ -40,71 +24,22 @@ def test_get_all():
     assert transforms.unified_scope[0] == "//Comment\nSELECT *;"
 
 
-@responses.activate
+@fake.json
 def test_replace_all():
-    # setup
-    project_json = utils.load_json("mastering_project.json")
-    project_url = tc.URL(path="projects/1")
-    responses.add(responses.GET, str(project_url), json=project_json)
-
-    tx_json = utils.load_json("transformations.json")
-    tx_url = tc.URL(path="projects/1/transformations")
-    responses.add(responses.GET, str(tx_url), json=tx_json)
-
-    dataset_json = utils.load_json("dataset.json")
-    dataset_url = tc.URL(path="datasets/1")
-    responses.add(responses.GET, str(dataset_url), json=dataset_json)
+    s = fake.session()
+    project = fake.mastering_project()
+    transforms = fake.transforms()
 
-    # test
-    s = utils.session()
-    instance = utils.instance()
-    project = tc.project.from_resource_id(s, instance, "1")
-
-    transforms = tc.transformations._from_json(s, instance, tx_json)
     transforms.unified_scope.append("//extra TX")
     transforms.input_scope.pop(1)
+    tc.transformations.replace_all(s, project, transforms)
 
-    responses.add(
-        responses.PUT, str(tx_url), json=tc.transformations._to_json(transforms)
-    )
-
-    r = tc.transformations.replace_all(s, project, transforms)
-
-    posted_tx = tc.transformations._from_json(s, project.url.instance, r.json())
```

Comment:
Author's note: the rest of this test function just tests serialization, which we already test via `test_get_all`. Users should only use the response returns from `tc.tranformations.replace_all` for debugging purposes. If users want the modified transformations, they should just call `tc.transformations.get_all` after.

Alternatively, we could modify `tc.transformations.replace_all` to return the modified.transformations